util/utility.js

```diff
@@ -347,7 +348,7 @@ const getData = fromPromise(async (redis, url) => {
               }
             }
             if (isMojangApi) {
-              throw new Error('Failed to get player uuid');
+              throw new PlayerError({ status: 500, message: 'Failed to get player uuid' });
```

Comment:
playerdb actually returns a 500 even when a player doesn't exists, the check on getUUID line 25 doesn't actually do anything

Reply:
About 30 minutes ago I was getting information about a player and Slothpixel returned a 404 error for:
![image](https://user-images.githubusercontent.com/42776347/112668772-9d535200-8eaa-11eb-9189-f335e7d18452.png)

This is hardcoded here:
https://github.com/slothpixel/core/blob/39753d5582679943ac8810e4648cdff171336f28/routes/spec.js#L366

I will delete 25 line in getUUID in the next commit.

Reply:
I doubt API should return normal Error if it is player related.

I will delete 25 line in getUUID in the next commit.